modules/index.d.ts

```diff
@@ -0,0 +1 @@
+export * from '../tslib.js'
```

Comment:
```suggestion
import tslib from '../tslib.js';
export * from '../tslib.js';
export default tslib;
```

Reply:
How’s this?

Reply:
Won't that depend on whether the consuming project's tsconfig.json has `esModuleIntrop: true` or `module: nodenext`? The _tslib.d.ts_ file doesn't specify a default export.

Reply:
If that's a concern, you could maybe do:

```ts
import * as tslib from '../tslib.js';
export default tslib;
```

or

```ts
export * as default from '../tslib.js';
```
?

Reply:
Well, only the combined `node` and `import` conditions point here, so the consuming project better have those things if they resolve to this file. But your suggestion is more portable, which might be useful if someone is doing something really weird.